pytest_tests/tests/object/test_object_api.py

```diff
@@ -736,6 +737,25 @@ def test_object_get_range_hash_negatives(
                             endpoint=self.neofs_env.sn_rpc,
                             range_cut=range_cut,
                         )
+                        
+    def test_put_object_header_limitation(
+        self, default_wallet: NodeWallet, container: str, simple_object_size: int
+    ):
+        file_path = generate_file(simple_object_size)
+        attr_key = "attr"
+        attr_val = "val"
+        attr_len = len(attr_key) + len(attr_val) + 9
```

Comment:
what `9` stands for?

Reply:
It was the "accurately" picked hardcoded number with the great sense in it :) I updated this.